packages/react-hot-loader/src/utils.dev.js

```diff
@@ -1,9 +1,9 @@
+import { setConfig as setProxyConfig } from 'react-stand-in'
 import { getProxyByType } from './reconciler/proxies'
 import reactHotLoader from './reactHotLoader'
 import logger from './logger'
-import { setConfig as setProxyConfig } from 'react-stand-in'
 
-setProxyConfig({ logger })
+setProxyConfig({ logger, reactHotLoader: reactHotLoader.config })
```

Comment:
If you are doing that, please remove `react-stand-in` and put all code in `react-hot-loader`. I think this is a good idea. We could also remove Lerna.

Reply:
Let me refactor this a bit. 
Anyway - this is a good decision to make.